### ISSUE TITLE
Installation on UEFI boot fails

### ISSUE THREAD

**Issue by ertankucukoglu** [NethServer/dev#6173]
First of all, I am not sure if this report belongs in here. Sorry for any inconvenience.

**Steps to reproduce**
- Find a system which can boot in UEFI.
- Use ISO from either NethServer 7.7.1098 or NethServer 7.8.2003

**Expected behavior**
- When GUI comes up, it should not have red text and automatic partitioning works.

**Actual behavior**
- When GUI comes up, it cannot automatically partition and it is not possible to start installation.

**Components**
Same problem exists for ISO versions of both versions above.

**See also**
https://community.nethserver.org/t/nethserver-installation-on-hyper-v-generation-2-aka-uefi/15568

----

Regular CentOS 7 netinstall works just fine.
I am using a virtual server for testing.

**Comment by DavidePrincipi** [comment]
**workaround (confirmed)**

Use the Manual installation method as described here:

https://community.nethserver.org/t/nethserver-installation-on-hyper-v-generation-2-aka-uefi/15568/17?u=davidep

**Comment by gsanchietti** [comment]
Thank you reporting!

It's been a while since I've tested UEFI installation, but I'm pretty sure it worked at least for NS 7.6.
As far as I know, there were no significant updates to anaconda from upstream which can cause a regression on this, but they also didn't fixed many existing bugs.

For the ISO, we added a couple of simple kickstart files which should already do the UEFI partition: see [relevant code](https://github.com/NethServer/nethserver-createiso/blob/master/src/lib/ks/unattended#L124).
See also the [official doc](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-syntax).

Since RHEL is more and more cloud-centric, I do not think they are going to fix anaconda for this usage scenario. Honestly I'd like to not spend more time on it since we have a viable workaround.

I think also the problem is relevant only for some UEFI implementation. If the problem is reproducible on all UEFI machines, we can add a note inside the manual.

I'm willing to close this as 'wontfix', if @DavidePrincipi  agrees.
